launcher/ui/pages/modplatform/ImportPage.cpp

```diff
@@ -148,6 +200,11 @@ void ImportPage::setUrl(const QString& url)
     updateState();
 }
 
+void ImportPage::setExtraInfo(const QMap<QString, QString>& extra_info) {
+    m_extra_info = QMap(extra_info); // copy
```

Comment:
the copy doesn't happen here, since QMap uses CoW (https://doc.qt.io/qt-6/qmap.html#QMap-4). This means that the comment is innacurate, and you don't need to wrap `extra_info` in `QMap(...)`.